src/db/seeds/0002-events.js

```diff
@@ -1,12 +1,17 @@
+const TABLE_NAME = 'Events';
+const HOUR_IN_MILLISECONDS = 60 * 60 * 1000;
+const DAY_IN_MILLISECONDS = 24 * HOUR_IN_MILLISECONDS;
+const roundToHour = (date) => new Date(Math.round(date.getTime() / HOUR_IN_MILLISECONDS) * HOUR_IN_MILLISECONDS);
+
 module.exports = {
-  up: (queryInterface, Sequelize) => queryInterface.bulkInsert('Events', [{
+  up: (queryInterface, Sequelize) => queryInterface.bulkInsert(TABLE_NAME, [{
     uuid: 'f9de28ca-80cc-4aeb-b8ed-ee9a92954e24',
     title: 'Bit-Byte Allocation',
     description: 'The big reveal is here!',
     committee: '',
     location: 'Qualcomm Room',
-    start: new Date(Date.now() - 86400000),
-    end: new Date(Date.now() - 82800000),
+    start: roundToHour(new Date(Date.now() - DAY_IN_MILLISECONDS)),
+    end: roundToHour(new Date(Date.now() - DAY_IN_MILLISECONDS - HOUR_IN_MILLISECONDS)),
```

Comment:
`Date.now() - DAY_IN_MILLISECONDS - HOUR_IN_MILLISECONDS` subtracts too much from the time.

Add parentheses around the constants:
`Date.now() - (DAY_IN_MILLISECONDS - HOUR_IN_MILLISECONDS)`